kitti_ros2bag_node:
  ros__parameters:
    base_dir: "/data/kitti/raw/2011_09_29"
    sequence: "2011_09_29_drive_0071_sync"  # synced+rectified data
    # this folder shall have image_00, image_01, image_02, image_03, oxts, and velodyne_points
    calib_folder: ""
    # by default the calibration files are under kitti_path. If you put the calibration files into
    # a different folder, then you will have to change it.
    # The folder shall have calib_cam_to_cam.txt, calib_imu_to_velo.txt, and calib_velo_to_cam.txt.
    # if not, the camera_info won't be outputted
